tools/methyldackel/MethylDackel.xml

```diff
@@ -104,301 +110,487 @@
+                </assert_contents>
+            </output> 
+        </test>
+
+        <!--Test keepStrand parameter-->
+        <test expect_num_outputs="1">
+            <param name="reference_source_selector" value="history" />
+            <param name="ref_file" value="cg100.fa" ftype="fasta" />
+            <param name="input_sortedAlignBAM" value="cg_aln.bam" ftype="bam"/>
+            <conditional name="regionBed_options">
+                <param name="regionBed_status" value="True"/>
+                <param name="regionBed" ftype="bed" value="cg100.bed"/>
+                <param name="keepStrand" value="True"/>
+            </conditional>
+            <conditional name="main_task">
+                <param name="task" value="extract" />
+            </conditional>
+            <section name="advanced_options">
+                <param name="min_mapq" value="2" />
+            </section>
+            <output name="outFileExtractCpG" ftype="bedgraph">
+                <assert_contents>
+                     <has_size value="846"/>
+                </assert_contents>
+            </output> 
+        </test>
+
+        <!-- Test mapability options-->
+        <test expect_num_outputs="1">
+            <param name="reference_source_selector" value="history" />
+            <param name="ref_file" value="cg100.fa" ftype="fasta" />
+            <param name="input_sortedAlignBAM" value="cg_aln.bam" ftype="bam"/>
+            <conditional name="main_task">
+                <param name="task" value="extract" />
+                <conditional name="mappability_options">
+                    <param name="mappability_status" value="True"/>
+                    <param name="mappability" value="mappability.bigwig" ftype="bigwig"/>
+                    <param name="mappabilityThreshold" value="0.1"/>
+                    <param name="minMappableBases" value="15"/>
+                </conditional>
+            </conditional>
+            <output name="outFileExtractCpG" ftype="bedgraph">
+                <assert_contents>
+                     <has_size value="66"/>
+                </assert_contents>
+            </output> 
+        </test>
+        <!-- Test cytosine report-->
+        <test expect_num_outputs="1">
+            <param name="reference_source_selector" value="history" />
+            <param name="ref_file" value="cg100.fa" ftype="fasta" />
+            <param name="input_sortedAlignBAM" value="cg_aln.bam" ftype="bam"/>
+            <conditional name="main_task">
+                <param name="task" value="extract" />
+                <param name="extract_outputs" value="--cytosine_report" />
+            </conditional>
+            <output name="outFileExtractCpG" ftype="txt">
+                <assert_contents>
+                     <has_size value="2147"/>
+                </assert_contents>
+            </output> 
         </test>
     </tests>
     <help><![CDATA[
 
-.. image:: $PATH_TO_IMAGES/MethylDackelLogo.png
```

Comment:
If you remove those images from the tool, can you also remove them from the tool dir?

Reply:
Sure, although this image did not exist previously.